src/core/blueprint/operators/http.rs

```diff
@@ -112,3 +155,36 @@ pub fn update_http<'a>(
         },
     )
 }
+
+/// extracts the keys from the json representation, if the value is of mustache
+/// template type.
+fn extract_expression_keys(json: &str) -> Vec<String> {
```

Comment:
- Make Body of Json Type
- Automatically get a dynamic value
- Iterate over the dynamic value and identify your key